buffet/src/combined/Inputs/index.js

```diff
@@ -83,7 +84,12 @@ function Inputs({
   }
 
   return (
-    <Error name={name} type={type} validations={validations}>
+    <Error
+      errors={errors[name] || false}
```

Comment:
The `errors` propType doesn't exist yet in the `Error` component. Is it a string or a bool or one of these types ?

Shouldn't be `errors[name] || null`  if it only can be a string, to prevent prop type errors ?

I think we may dissociate the `error`state to the `error` prop in the `Error` compo because the children is a node displaying a string.

So the `Description` and `ErrorMessage` display below should be a bit different if we don't extend the `translatedErrors` prop type.